etc/adminhtml/system.xml

```diff
@@ -334,6 +334,35 @@
                         </depends>
                     </field>
                 </group>
+                <group id="omise_offsite_fpx" translate="label" sortOrder="400" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable FPX Payment</label>
+                    <comment>Enable customers to pay via FPX</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="410" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
+                        <comment>Enable FPX payment method.</comment>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/omise_offsite_fpx/active</config_path>
+                    </field>
+                    <field id="title" translate="label" type="text" sortOrder="420" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Title</label>
+                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <config_path>payment/omise_offsite_fpx/title</config_path>
+                    </field>
+                    <field id="allowspecific" translate="label comment" type="select" sortOrder="421" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Allowed Countries</label>
+                        <config_path>payment/omise_offsite_fpx/allowspecific</config_path>
+                        <source_model>Magento\Payment\Model\Config\Source\Allspecificcountries</source_model>
+                        <comment>If set to specific, guest customers will not have a billing country and may not be able to checkout.</comment>
```

Comment:
Can this be rephrased? e.g. "If specified, ..."

Reply:
Should i change this for other payments method as well?

Reply:
I think so, but I'm going to amend my suggestion to "If set to specific countries, ..."

Reply:
Let's confirm the wording tomorrow morning first

Reply:
I think the best wording in the end is "If not set to all, ..."